lib/config/presets/internal/replacements.ts

```diff
@@ -51,6 +52,18 @@ export const presets: Record<string, Preset> = {
     ],
     ignoreDeps: [], // Hack to improve onboarding PR description
   },
+  'airbnb-prop-types-to-prop-types-tools': {
+    '`airbnb-prop-types` was given to a new maintainer and renamed',
```

Comment:
```suggestion
    description: '`airbnb-prop-types` was given to a new maintainer and renamed to `prop-types-tools`.',
```

Changes:

- Add `description:` part before the description
- End description sentence with a full stop: `.`
- Add name of replacement package to the description

Reply:
oh duh thanks